eos-installer-data/meson.build

```diff
@@ -0,0 +1,35 @@
+desktop_conf = configuration_data()
+desktop_conf.set('LIBEXECDIR', libexecdir)
+
+i18n.merge_file(
+    input: configure_file(
+        input: files('com.endlessm.Installer.desktop.in.in'),
+        output: 'com.endlessm.Installer.desktop.in',
+        configuration: desktop_conf
+    ),
+    output: 'com.endlessm.Installer.desktop',
+    install_dir: join_paths(datadir, 'applications'),
+    po_dir: '../po',
+    install: true,
+    type: 'desktop'
+)
+
+data_conf = configuration_data()
+data_conf.set('libexecdir', libexecdir)
+
+install_data(
+    'run-mount-eosimages.mount',
+    install_dir: systemdsystemunitdir,
+)
+
+rules_dir = join_paths(datadir, 'polkit-1', 'rules.d')
+install_data(
+    '90-eos-installer.rules',
+    install_dir: rules_dir,
+)
+
+icon_dir = 'icons/hicolor/64x64/apps'
+install_data(
+    icon_dir / 'com.endlessm.Installer.png',
+    install_dir: datadir / icon_dir,
+)
```

Comment:
No newline at end of file.

Reply:
Guess which one I wrote with GNOME Builder (defaults to no trailing newline) and which with Vim!

`.editorconfig` added.

Reply:
Why would builder default to no trailing newline? :roll_eyes: